mkdocs.yml

```diff
@@ -245,7 +245,8 @@ plugins:
       raise_error: true
       raise_error_excludes:
         404: ['https://support.google.com/accounts/answer/185833?hl=en',
-              'https://support.google.com/accounts/answer/6010255?hl=en']
+              'https://support.google.com/accounts/answer/6010255?hl=en',
+              'https://docs.docker.com/compose/extends/#multiple-compose-files']
```

Comment:
I just verify this link is working again, so we can now remove it from this file.

Reply:
Thanks Lenny, updated